float32 fb_speed
float32 yaw
int32 gear
bool brake